test/db.tests.js

```diff
@@ -272,6 +272,7 @@ module.exports.tests = (clientCreator) => {
                   assert.ok(result.conformant);
                   assert.equal(result.remaining, 9);
                   assert.closeTo(result.reset, now / 1000, 3);
+                  assert.closeTo(result.delta_reset_ms, (result.limit - result.remaining) * 1000/buckets.ip.per_second, 3);
```

Comment:
no. It's giving some room to avoid flaky tests. Might actually not be enough. In this case I'm saying delta_reset_ms should be within 3ms of my calculations, but depends on how fast the test runs.